[gdal] Build error

**Host Environment**
- OS: [Windows11 x64]
- Compiler: mingw 8.1.0 64bit


**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg.exe install gdal --triplet=x64-mingw-dynamic`

**Failure logs**

```
$ ./vcpkg.exe install gdal  --triplet=x64-mingw-dynamic
Computing installation plan...
The following packages will be built and installed:
    gdal[core,hdf5,netcdf,postgresql,supported-default-features]:x64-mingw-dynamic -> 3.4.3
Detecting compiler hash for triplet x64-mingw-dynamic...
Restored 0 packages from C:\Users\o\AppData\Local\vcpkg\archives in 170.8 us. Use --debug to see more details.
Installing 1/1 gdal:x64-mingw-dynamic...
Building gdal[core,hdf5,netcdf,postgresql,supported-default-features]:x64-mingw-dynamic...
-- Using community triplet x64-mingw-dynamic. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: G:\buildtmp\vcpkg\triplets\community\x64-mingw-dynamic.cmake
-- Using cached OSGeo-gdal-v3.4.3.tar.gz.
-- Cleaning sources at G:/buildtmp/vcpkg/buildtrees/gdal/src/v3.4.3-d31eaa7065.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source G:/buildtmp/vcpkg/downloads/OSGeo-gdal-v3.4.3.tar.gz
-- Applying patch 0001-Fix-debug-crt-flags.patch
-- Applying patch 0002-Fix-build.patch
-- Applying patch 0004-Fix-cfitsio.patch
-- Applying patch 0005-Fix-configure.patch
-- Applying patch 0006-Fix-mingw-dllexport.patch
-- Applying patch 0007-Control-tools.patch
-- Applying patch 0008-Fix-absl-string_view.patch
-- Applying patch 0009-atlbase.patch
-- Applying patch 0010-symprefix.patch
-- Using source at G:/buildtmp/vcpkg/buildtrees/gdal/src/v3.4.3-d31eaa7065.clean
-- Getting CMake variables for x64-mingw-dynamic
-- Using cached msys-gzip-1.11-1-x86_64.pkg.tar.zst.
-- Using cached msys-bash-5.1.008-1-x86_64.pkg.tar.zst.
-- Using cached msys-autoconf-2.71-3-any.pkg.tar.zst.
-- Using cached msys-diffutils-3.8-2-x86_64.pkg.tar.zst.
-- Using cached msys-binutils-2.37-5-x86_64.pkg.tar.zst.
-- Using cached msys-libtool-2.4.6-9-x86_64.pkg.tar.xz.
-- Using cached msys-file-5.41-2-x86_64.pkg.tar.zst.
-- Using cached msys-zlib-1.2.11-1-x86_64.pkg.tar.xz.
-- Using cached msys-libbz2-1.0.8-3-x86_64.pkg.tar.zst.
-- Using cached msys-coreutils-8.32-2-x86_64.pkg.tar.zst.
-- Using cached msys-grep-3.0-2-x86_64.pkg.tar.xz.
-- Using cached msys-sed-4.8-2-x86_64.pkg.tar.zst.
-- Using cached msys-libpcre-8.45-1-x86_64.pkg.tar.zst.
-- Using cached msys-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys-automake-wrapper-11-4-any.pkg.tar.zst.
-- Using cached msys-gawk-5.1.0-2-x86_64.pkg.tar.zst.
-- Using cached msys-mpfr-4.1.0-1-x86_64.pkg.tar.zst.
-- Using cached msys-gmp-6.2.1-1-x86_64.pkg.tar.zst.
-- Using cached msys-libreadline-8.1.001-1-x86_64.pkg.tar.zst.
-- Using cached msys-ncurses-6.2-2-x86_64.pkg.tar.zst.
-- Using cached msys-automake1.16-1.16.3-3-any.pkg.tar.zst.
-- Using cached msys-perl-5.32.1-2-x86_64.pkg.tar.zst.
-- Using cached msys-libcrypt-2.1-3-x86_64.pkg.tar.zst.
-- Using cached msys-pkg-config-0.29.2-4-x86_64.pkg.tar.zst.
-- Using cached msys-make-4.3-3-x86_64.pkg.tar.zst.
-- Using cached msys-findutils-4.8.0-1-x86_64.pkg.tar.zst.
-- Using cached msys-libintl-0.21-1-x86_64.pkg.tar.zst.
-- Using cached msys-libiconv-1.16-2-x86_64.pkg.tar.zst.
-- Using cached msys-gcc-libs-11.2.0-3-x86_64.pkg.tar.zst.
-- Using cached msys-msys2-runtime-3.2.0-8-x86_64.pkg.tar.zst.
-- Using msys root at G:/buildtmp/vcpkg/downloads/tools/msys2/7e05e7aa09f1709f
-- Generating configure for x64-mingw-dynamic
-- Finished generating configure for x64-mingw-dynamic
-- Using cached msys-mingw-w64-i686-pkg-config-0.29.2-3-any.pkg.tar.zst.
-- Using cached msys-mingw-w64-i686-libwinpthread-git-9.0.0.6373.5be8fcd83-1-any.pkg.tar.zst.
-- Using msys root at G:/buildtmp/vcpkg/downloads/tools/msys2/9a1ec3f33446b195
-- Configuring x64-mingw-dynamic-rel
-- Building x64-mingw-dynamic-rel
CMake Error at scripts/cmake/vcpkg_build_make.cmake:212 (message):
  libtool could not find a file being linked against!
Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_make.cmake:26 (vcpkg_build_make)
  ports/gdal/portfile.cmake:326 (vcpkg_install_make)
  scripts/ports.cmake:146 (include)


error: building gdal:x64-mingw-dynamic failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+gdal
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[gdal]+Build+error
Include '[gdal] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-05-05-67e17c1782801cf481be9ac0b3765dff3e4bdeb8
    vcpkg-scripts version: 93b38546e 2022-05-24 (13 hours ago)
```

I don't understand, why will I compile the i686 version here?
> The folder `G:/buildtmp/vcpkg/downloads/tools/msys2/7e05e7aa09f1709f` is mingw64 tool chain.
> `G:/buildtmp/vcpkg/downloads/tools/msys2/9a1ec3f33446b195` is the incomplete mingw32.

> What you call "mingw32" just serves to provide `pkg-config` on Windows.
> 
> Please provide:
> 
> ```
> buildtrees/gdal/build-x64-mingw-dynamic-rel-err.log
> buildtrees/gdal/build-x64-mingw-dynamic-rel-out.log
> ```

**buildtrees/gdal/build-x64-mingw-dynamic-rel-err.log**
```bash
cpl_vsil_win32.cpp: In function 'const char* CPLGetWineVersion()':
cpl_vsil_win32.cpp:454:107: warning: cast between incompatible function types from 'FARPROC' {aka 'long long int (*)()'} to 'const char* (*)()' [-Wcast-function-type]
     pwine_get_version = reinterpret_cast<const char* (*)(void)>(GetProcAddress(hntdll, "wine_get_version"));
                                                                                                           ^
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_curl_streaming.cpp:32:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_curl_streaming.cpp:32:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_curl.cpp:31:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_curl.cpp:31:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_gs.cpp:33:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_gs.cpp:33:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_s3.cpp:37:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_s3.cpp:37:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_az.cpp:34:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_az.cpp:34:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_adls.cpp:34:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_adls.cpp:34:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_oss.cpp:33:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_oss.cpp:33:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_swift.cpp:34:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_swift.cpp:34:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_vsil_curl_class.h:35,
                 from cpl_vsil_webhdfs.cpp:33:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_aws.h:43,
                 from cpl_vsil_curl_class.h:34,
                 from cpl_vsil_webhdfs.cpp:33:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from cpl_azure.cpp:28:
cpl_azure.h:54:13: warning: declaration of 'BLOB' shadows a global declaration [-Wshadow]
             BLOB,
             ^~~~
In file included from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/combaseapi.h:153,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/objbase.h:14,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/ole2.h:17,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypes.h:12,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winscard.h:10,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/windows.h:97,
                 from C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/winsock2.h:23,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/system.h:406,
                 from G:/buildtmp/vcpkg/installed/x64-mingw-dynamic/include/curl/curl.h:35,
                 from cpl_azure.h:35,
                 from cpl_azure.cpp:28:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/wtypesbase.h:385:3: note: shadowed declaration is here
 } BLOB;
   ^~~~
In file included from ogrodsdatasource.cpp:34:
ods_formula.h:41: warning: "strcasecmp" redefined
 #  define strcasecmp stricmp
 
In file included from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_port.h:141,
                 from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_progress.h:33,
                 from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/ogr/ogrsf_frmts/ogrsf_frmts.h:34,
                 from ogr_ods.h:33,
                 from ogrodsdatasource.cpp:29:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/string.h:120: note: this is the location of the previous definition
 #define strcasecmp _stricmp
 
netcdfdataset.cpp: In function 'CPLErr NCDFGetSubGroups(int, int*, int**)':
netcdfdataset.cpp:11497:37: warning: unused parameter 'nGroupId' [-Wunused-parameter]
 static CPLErr NCDFGetSubGroups( int nGroupId, int *pnSubGroups,
                                 ~~~~^~~~~~~~
netcdfdataset.cpp: In function 'CPLErr NCDFGetGroupFullName(int, char**, bool)':
netcdfdataset.cpp:11520:41: warning: unused parameter 'nGroupId' [-Wunused-parameter]
 static CPLErr NCDFGetGroupFullName( int nGroupId, char **ppszFullName,
                                     ~~~~^~~~~~~~
netcdfdataset.cpp: In function 'bool NCDFIsUserDefinedType(int, int)':
netcdfdataset.cpp:12270:32: warning: unused parameter 'ncid' [-Wunused-parameter]
 bool NCDFIsUserDefinedType(int ncid, int type)
                            ~~~~^~~~
netcdfdataset.cpp:12270:42: warning: unused parameter 'type' [-Wunused-parameter]
 bool NCDFIsUserDefinedType(int ncid, int type)
                                      ~~~~^~~~
In file included from ods_formula.cpp:34:
ods_formula.h:41: warning: "strcasecmp" redefined
 #  define strcasecmp stricmp
 
In file included from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_port.h:141,
                 from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_conv.h:35,
                 from ods_formula.cpp:33:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/string.h:120: note: this is the location of the previous definition
 #define strcasecmp _stricmp
 
In file included from ods_formula_node.cpp:32:
ods_formula.h:41: warning: "strcasecmp" redefined
 #  define strcasecmp stricmp
 
In file included from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_port.h:141,
                 from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_conv.h:35,
                 from ods_formula_node.cpp:31:
C:/Qt/Tools/mingw810_64/x86_64-w64-mingw32/include/string.h:120: note: this is the location of the previous definition
 #define strcasecmp _stricmp
 
In file included from G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_conv.h:35,
                 from ogrosmlayer.cpp:41:
ogrosmlayer.cpp: In member function 'void OGROSMLayer::SetFieldsFromTags(OGRFeature*, GIntBig, bool, unsigned int, OSMTag*, OSMInfo*)':
G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_port.h:312:26: warning: format '%d' expects argument of type 'int', but argument 4 has type 'GIntBig' {aka 'long long int'} [-Wformat=]
 #define CPL_FRMT_GIB     "%" CPL_FRMT_GB_WITHOUT_PREFIX "d"
                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ogrosmlayer.cpp:603:42: note: in expansion of macro 'CPL_FRMT_GIB'
             snprintf(szID, sizeof(szID), CPL_FRMT_GIB, nID );
                                          ^~~~~~~~~~~~
G:/buildtmp/vcpkg/buildtrees/gdal/x64-mingw-dynamic-dbg/port/cpl_port.h:312:26: warning: format '%d' expects argument of type 'int', but argument 4 has type 'GIntBig' {aka 'long long int'} [-Wformat=]
 #define CPL_FRMT_GIB     "%" CPL_FRMT_GB_WITHOUT_PREFIX "d"
                          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ogrosmlayer.cpp:614:42: note: in expansion of macro 'CPL_FRMT_GIB'
             snprintf(szID, sizeof(szID), CPL_FRMT_GIB, nID );
                                          ^~~~~~~~~~~~
```
[build-x64-mingw-dynamic-dbg-err.log](https://github.com/microsoft/vcpkg/files/8776658/build-x64-mingw-dynamic-dbg-err.log)
[build-x64-mingw-dynamic-dbg-out.log](https://github.com/microsoft/vcpkg/files/8776659/build-x64-mingw-dynamic-dbg-out.log)
```

--- a/ports/gdal/0005-Fix-configure.patch
+++ b/ports/gdal/0005-Fix-configure.patch
@@ -11,6 +11,25 @@ index 30cd613..43f1daf 100644
  dnl Enable as much warnings as possible
  AC_LANG_PUSH([C])
  AX_COMPILER_VENDOR
+@@ -1111,6 +1111,8 @@ dnl ---------------------------------------------------------------------------
+ AC_ARG_WITH(libz,[  --with-libz[=ARG]       Include libz support (ARG=internal or libz directory)],,)
+ 
+ if test "$with_libz" = "external" -o "$with_libz" = "" -o "$with_libz" = "yes" ; then
++  PKG_CHECK_MODULES([ZLIB], [zlib], [LIBZ_SETTING=external], [LIBZ_SETTING=internal])
++elif false ; then
+ 
+   AC_CHECK_LIB(z,deflateInit_,LIBZ_SETTING=external,LIBZ_SETTING=internal,)
+ 
+@@ -1144,7 +1146,8 @@ elif test "$with_libz" != "no" -a "$with_libz" != "internal" ; then
+ fi
+ 
+ if test "$LIBZ_SETTING" = "external" ; then
+-  LIBS="-lz $LIBS"
++  LIBS="${ZLIB_LIBS} $LIBS"
++  EXTRA_INCLUDES="${ZLIB_CFLAGS} $EXTRA_INCLUDES"
+   if test "$with_libz" != "" -a "$with_libz" != "yes" -a "$with_libz" != "external" ; then
+     EXTRA_INCLUDES="-I$with_libz -I$with_libz/include $EXTRA_INCLUDES"
+   fi
 @@ -1336,12 +1338,15 @@ AC_MSG_CHECKING([for libtiff])
  
  if test "x${with_libtiff}" = "xyes" -o "x${with_libtiff}" = "x" ; then

--- a/ports/gdal/vcpkg.json
+++ b/ports/gdal/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "gdal",
   "version-semver": "3.4.3",
+  "port-version": 1,
   "description": "The Geographic Data Abstraction Library for reading and writing geospatial raster and vector data",
   "homepage": "https://gdal.org",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2434,7 +2434,7 @@
     },
     "gdal": {
       "baseline": "3.4.3",
-      "port-version": 0
+      "port-version": 1
     },
     "gdcm": {
       "baseline": "3.0.12",

--- a/versions/g-/gdal.json
+++ b/versions/g-/gdal.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8bdf8b8ff951c1ce8719c789e66ac3a166eed063",
+      "version-semver": "3.4.3",
+      "port-version": 1
+    },
+    {
       "git-tree": "8277225a2c5da40e154062f73526126fb00c9125",
       "version-semver": "3.4.3",
       "port-version": 0

